server/src/api/users.ts

```diff
@@ -152,6 +183,35 @@ router.post("/users/changepassword", authCheck, async (req, res, next) => {
   }
 });
 
+/*
+* Invite a new user via email
+*/
+router.post("/users/invite", authCheck, async (req, res, next) => {
```

Comment:
Is there email validation and checking if email already exists?

Reply:
There is not - will fix